test/IRGen/report_dead_method_call.swift

```diff
@@ -5,6 +5,9 @@
 // RUN: %target-run %t/a.out p1 p2 2> %t/err3.log; FileCheck %s < %t/err3.log
 // REQUIRES: executable_test
 
+// This test correctly outputs "fatal error" on an Android device, but
+// the Android test runner interprets the abort as a test failure.
```

Comment:
Could you rewrite it using StdlibUnitest then?  It has special provisions for detecting crashes.